Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,22 +236,25 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
```

Comment:
```suggestion
```

Reply:
I think the `puts` show stay too. It helps with clarity.